ocs_ci/ocs/utils.py

```diff
@@ -1460,7 +1460,7 @@ def reboot_node(ceph_node, timeout=300):
         raise
 
 
-def enable_console_plugin():
+def enable_console_plugin(value="[odf-console]"):
```

Comment:
not clear what "value" argument means. It seems we don't need it, bacause the method has only one purpose - to enable` odf-console`
Please remove this argument from method

Reply:
This fuction can be used to enable odf console and odf client console for storageclients.

Reply:
makes sense. 
in this case please add to a docstring what is the 'value' and consider renaming it to a meaningful argument name

Reply:
Updated